vNext/test/NonSiloTests/SerializationTests/ConfigurationTests/ClientConfigurationForSerializer.xml

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ClientConfiguration xmlns="urn:orleans">
+  <Gateway Address="localhost" Port="40000"/>
+  <!-- To turn tracing off, set DefaultTraceLevel="Off" and have no overrides.
+    For the trace log file name, {0} is replaced by "Client" and {1} is the current time. -->
+  <Tracing DefaultTraceLevel="Info" TraceToConsole="false" TraceToFile="{0}-{1}.log" BulkMessageLimit="1000">
+  </Tracing>
+  <Statistics MetricsTableWriteInterval="300s" PerfCounterWriteInterval="30s" LogWriteInterval="300s" WriteLogStatisticsToTable="true" StatisticsCollectionLevel="Info"/>
+  <Messaging ResponseTimeout="30s" ClientSenderBuckets="8192" MaxResendCount="0">
+    <SerializationProviders>
+      <Provider type="UnitTests.Serialization.FakeSerializer, Orleans.NonSiloTests"/>
+    </SerializationProviders>
```

Comment:
I think I miss-understood this a little bit, and will be a lot more common (a ton more of these files) when we port silo tests in the upcoming days. I guess we'll start deviating a lot if we copy these files.
I think our best alternatives are:
1. Create an Orleans.NonSiloTests project in the main solution (which I believe it's valuable regardless)
2. port them to another project which share the same assembly name as main solution. (what you suggested before).

I believe 1 is best, what do you think?